apps/prairielearn/elements/pl-order-blocks/pl-order-blocks.css

```diff
@@ -51,6 +52,11 @@ li.ui-state-highlight {
   margin-bottom: 0px;
 }
 
+div.pl-order-blocks-container {
+  display: flex;
+  flex-direction: column;
+}
+
```

Comment:
This should go into the `div.pl-order-blocks-container` declaration below so that we don't have two seperate declarations for the same class.

Reply:
Good catch, thank you.